scripts/migrations/migration_155.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# * coding: utf8 *
+'''
+migration_155.py
+
+A module that adjusts scale and precision on some float fields
+'''
+
+from os.path import dirname, join
+
+import arcpy
+
+arcpy.env.workspace = sde = join(dirname(__file__), 'local@electrofishing.sde')
+
+
+def recreate_field(table, field_infos):
+    fields = [field[0] for field in field_infos]
+    print('deleting {} from {}'.format(fields, table))
+    arcpy.management.DeleteField(table, fields)
+
+    for field_name, alias, precision, scale in field_infos:
+        print('adding {}'.format(field_name))
+        arcpy.management.AddField(table, field_name=field_name, field_type='FLOAT', field_precision=precision, field_scale=scale, field_alias=alias)
+
+
+table_name = 'Equipment'
+field_infos = [
+    ('VOLTAGE', 'Voltage', 5, 6),  #: min 0, max 1000, steps .01
```

Comment:
Shouldn't this be precision = 6 and scale = 2? Or am I crazy?

Reply:
Oops

Reply:
Ladies and Gents: He's human.